### ISSUE TITLE
Support for Angular 13

### ISSUE THREAD

**Issue by parse** [apexcharts/ng-apexcharts#171]
As Angular 13 has been released, is it possible to release a new version with updated dependencies? I've been running it using -force flag and it seems to work.

Thanks!

**Comment by tuuling** [comment]
@junedchhipa could you provide an estimate on when we could get this? 
Tests are broken on Angular 13 without Ivy support. I'd rather wait to get this package updated - otherwise I would have to stub it in tests.

**Comment by bgpedersen** [comment]
We tried using --force, but we get a lot of Bad Engine errors. So we are currently blocked untill ng-apexcharts supports Angular 13 with RxJs 7. 

We are looking much forward for it 😄

**Comment by chalkplum** [comment]
any update on this?

**Comment by jeoskiploop** [comment]
Still getting same issue on angular 13. Any update on this?

**Comment by jeosebastian555** [comment]
**I have used --force to install the package and it worked.** And once I tried the samples provided I was getting some undefined errors. I could resolve it by changing from 

```
 public chartOptions: Partial<ChartOptions>;
 constructor() {
    this.chartOptions = { }
 }
```

to

```
public chartOptions:ChartOptions = { 
  //provide all the keys here instead of assigning values in constructor
}
```

The partial keyword is not required. Also in sample code below code snippet is not used anywhere so I removed it.

`@ViewChild("chart") chart: ChartComponent;`

Thank you.

**Comment by bgpedersen** [comment]
Any updates on supporting Angular 13 ?

**Comment by jmmartinl** [comment]
> **I have used --force to install the package and it worked.** And once I tried the samples provided I was getting some undefined errors. I could resolve it by changing from
> 
> ```
>  public chartOptions: Partial<ChartOptions>;
>  constructor() {
>     this.chartOptions = { }
>  }
> ```
> 
> to
> 
> ```
> public chartOptions:ChartOptions = { 
>   //provide all the keys here instead of assigning values in constructor
> }
> ```
> 
> The partial keyword is not required. Also in sample code below code snippet is not used anywhere so I removed it.
> 
> `@ViewChild("chart") chart: ChartComponent;`
> 
> Thank you.

This solution works also perfectly for me in Angular 13. Thanks

**Comment by Jalgoczy** [comment]
Any updates? Is there a new version coming out soon that supports Angular 13?

Thanks.

**Comment by morbitech1** [comment]
Also a showstopper for me, would love to see this go through!

**Comment by contact2vikas** [comment]
Any updates on supporting Angular 13 ?

**Comment by junedchhipa** [comment]
Released v1.7.0 with Angular 13 support
Thanks @damingerdai
